inspire/base/templates/header.html

```diff
@@ -32,355 +32,342 @@
-                    display: inline-block;
-                    margin-top: 20px;
-                    margin-right: 60px;
-                    color: #000;">
-                    <div class="search-box" style="display: inline-block;">
-                        <input type="text" name="p" placeholder="Search INSPIRE Labs" value="" style="width: 200px;">
-                    </div>
-                    <div class="search-dropdown" style="display: inline-block;">
-                    <select name="cc">
-                      <option value="HEP">HEP</option>
-                      <option value="HepNames">Authors</option>
-                      <option value="Institutions">Institutions</option>
-                      <option value="Conferences">Conferences</option>
-                      <option value="Jobs">Jobs</option>
-                      <option value="Experiments">Experiments</option>
-                      <option value="Journals">Journals</option>
-                    </select>
-                    </div>
-                    <button type="submit" class="btn-warning search-submit">Search</button></form>
-                  </li>
-                {% endif %}
-              <!-- Nice dropdown  -->
-               <li class="dropdown">
-                <a class="dropdown-toggle" data-toggle="dropdown"  href="#">Resources</a>
-                <ul class="dropdown-menu dropdown-menu-amazon" id="dropdown-menu-amazon" role="menu">
-                  <li data-submenu-id="jobs" class="first-submenu-item">
-                   <div class="row">
-                    <div class="col-md-8"></div>
-                    <div id="first-element" class="col-md-4 cursor-pointer small-screens dropdown-menu-list">
-                      <span class="dropdown-options">
-                        <a class="text-link-decoration small-screens-a"><i class="fa fa-briefcase"></i> Jobs</a>
-                      </span>
-                        <i id="first-arrow" class="fa fa-chevron-right arrow-position"></i>
-                    </div>
+{% if collection is defined %}
+  {% set collection_name = collection.name %}
+{% elif collection_name is not defined %}
+  {% set collection_name = '' %}
+{% endif %}
+
+<section>
+  <nav id="topnav" class="navbar navbar-fixed-top navbar-inverse navbar-custom" role="navigation">
+    <div class="container">
+      <div class="navbar-header">
+        <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target=".navbar-ex1-collapse">
+          <span class="sr-only">{{ _("Toggle navigation") }}</span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </button>
+        <a class="navbar-brand" href="{{ url_for('collections.index') }}">
+          <span id="{{ collection_name | sanitize_collection_name }}-collection">
+            {% include 'labs-logo.svg' %}
```

Comment:
@jmartinm I've moved the logo file to `/templates` because it seems like I can't include it from any other folder over here. Is that valid?

The logo needs to be output this way in order to have access through CSS to the elements of the SVG file, in order to change the colors of the logo.